doc/neps/nep-0030-duck-array-protocol.rst

```diff
@@ -90,7 +96,8 @@ a complete implementation would look like the following:
             return self
 
         def __array__(self):
-            return TypeError
+            return TypeError("NumPyLikeArray can not be converted to a numpy "
```

Comment:
This should raise a TypeError, not return one. Do we use `NumPyLikeArray` above? I would use either `DuckArray` or `ArrayLike`.

Otherwise seems good. @pentschev do you want to sign the changes off, since it is your NEP?

Reply:
Ah, we had some other fixups in this file, is this already in, can you rebase? (I guess that might be annoying in this case...)

Reply:
This is a great catch @seberg! I think the `return` statement has been there since the initial proposal and nobody catched it before.

Regarding `NumPyLikeArray`, that's the name of the example class. Since it's just really an example, I particularly don't care if the name remains the same or we change it to one of the proposed suggestions. In general, it has been difficult to find good names, since many of them end up being ambiguous.